data/json/vitrification/vitrified_snippets.json

```diff
@@ -0,0 +1,145 @@
+[
+  {
+    "type": "snippet",
+    "category": "Vitrifaction_Threshold",
+    "text": [
+      "You arrive at the farmhouse, after what feels like an eternity. It's not quite what you were expecting, but it's very peaceful."
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "Vitrifaction_Blocked",
+    "text": [ "There's nothing that way but a plane of featureless black." ]
+  },
+  {
+    "type": "snippet",
+    "category": "Vitrifaction_Ambient_1",
+    "text": [
+      "With all this glass, you expected it to be brighter here. It's not. That's okay, it would have hurt your eyes otherwise.",
+      "A gust of wind blows across the farm. Glass chimes and keens in the breeze.",
+      "You are suddenly very aware of your heartbeat, but only for a moment.",
+      "The glass around you isn't sparkling; it doesn't reflect. You're not sure what it's doing instead.",
+      "It's very cold here.",
+      "You didn't realize there had been an ache at the base of your neck all day until just now, when you notice it's gone.",
+      "There's nothing here, really. How long has it been since you've felt truly safe?",
+      "You find yourself wondering what the glass tastes like.",
+      "The glass is solid under your feet. It supports you completely. Why can't you feel yourself standing?"
```

Comment:
```suggestion
      "With all this glass, you expected it to be brighter here.  It's not.  That's okay, it would have hurt your eyes otherwise.",
      "A gust of wind blows across the farm.  Glass chimes and keens in the breeze.",
      "You are suddenly very aware of your heartbeat, but only for a moment.",
      "The glass around you isn't sparkling; it doesn't reflect. You're not sure what it's doing instead.",
      "It's very cold here.",
      "You didn't realize there had been an ache at the base of your neck all day until just now, when you notice it's gone.",
      "There's nothing here, really.  How long has it been since you've felt truly safe?",
      "You find yourself wondering what the glass tastes like.",
      "The glass is solid under your feet. It supports you completely.  Why can't you feel yourself standing?"
```